### DISCUSSION TITLE
Rainmeter & CorsairLink

### DISCUSSION THREAD

**Discussion by zeldasdad** [EvanMulawski/FanControl.CorsairLink#36] [Q&A] [answered]
Not super closely related to our beloved FanControl Corsair plugin, but ... anyone have any experience tying into Corsair statistics with Rainmeter? Could I tap into say, this plugin's log for RPM data?

Thanks!

**Comment by EvanMulawski** [comment, marked as answer]
Hi @tiltedkilt, HWiNFO can be used concurrently with the CorsairLink plugin so you should be able to use Rainmeter's HWiNFO integration directly. Let me know if you run into any issues.

**Comment by zeldasdad** [comment]
You're exactly right. I was under the impression that HWiNFO wouldn't report RPM with FanControl running, but it does, I'm happy to report. In other news, commander core xt is terrible, and I'm getting rid of it. HWiNFO only recently added support for it, and even with that, it has collisions with the built-in LED lighting node. I've seen the code you've written to deal with the commander core, I'm getting the heck out of dodge with that device.
